spikeinterface/core/curationsorting.py

```diff
@@ -0,0 +1,119 @@
+from .mergesorting import MergeSorting
+from .splitsortingunit import SplitSortingUnit
+import networkx as nx
+from collections import namedtuple
+node_t = namedtuple('node_t', 'id stage')
+
+class CurationSorting():
+    """
+    Class that handles curation of a Sorting object.
+
+    """
+
+    def __init__(self, parent_sorting):
```

Comment:
We could inherit from `BaseSorting` too. I think it would make it convenient so that all functions would be directly available on the top level. All we need to do is instantiate the object with the initial sorting unit ids, and then override `get_unit_ids`, `get_unit_spike_train`, and `get/set_property`. Or (but it needs to be tested), we can do something like: 
```
self = self._sorting_stages[self._sorting_stages_i]
```
and we have to call this whenever an action is made. Let's discuss about this!

Reply:
the issue with inherit from BaseSorting is that we will need to keep updating the functions if we change something in the BaseSorting 

In a way curationsorting is not a proper BaseSorting, but a basesorting handler. Using the methods as a BaseSorting  is just for simplicity, I'm not even sure if using the `sorting` property to access the current sorting is enough